docs/cli.md

```diff
@@ -91,68 +103,96 @@ model, the Model Analyzer will save the measurements it has collected into the
 
 ### Examples
 
-Some example profile commands are shown here. For a full example see the 
+Some example profile commands are shown here. For a full example see the
 [quick start](./quick_start.md) section.
 
 1. Run auto config search on a model called `resnet50_libtorch` located in `/home/model_repo`
 
-  ```
-  $ model-analyzer profile -m /home/model_repo --profile-models resnet50_libtorch
-  ```
+```
+$ model-analyzer profile -m /home/model_repo --profile-models resnet50_libtorch
+```
 
 2. Run auto config search on 2 models called `resnet50_libtorch` and `vgg16_graphdef` located in `/home/model_repo` and save checkpoints to `checkpoints`
 
-  ```
-  $ model-analyzer profile -m /home/model_repo --profile-models resnet50_libtorch,vgg16_graphdef --checkpoint-directory=checkpoints
-  ```
+```
+$ model-analyzer profile -m /home/model_repo --profile-models resnet50_libtorch,vgg16_graphdef --checkpoint-directory=checkpoints
+```
 
 3.  Run auto config search on a model called `resnet50_libtorch` located in `/home/model_repo`, but change the repository where model config variants are stored to `/home/output_repo`
 
-  ```
-  $ model-analyzer profile -m /home/model_repo --output-model-repository-path=/home/output_repo --profile-models resnet50_libtorch
-  ```
+```
+$ model-analyzer profile -m /home/model_repo --output-model-repository-path=/home/output_repo --profile-models resnet50_libtorch
+```
 
 4. Run profile over manually defined configurations for a models `classification_malaria_v1` and `classification_chestxray_v1` located in `/home/model_repo` using the YAML config file
 
-  ```
-  $ model-analyzer profile -f /path/to/config.yaml
-  ```
-
 The contents of `config.yaml` are shown below.
-```yaml
 
+```yaml
 model_repository: /home/model_repo
 
 run_config_search_disable: True
 
-concurrency: [2,4,8,16,32]
-batch_sizes: [8,16,64]
+concurrency: [2, 4, 8, 16, 32]
+batch_sizes: [8, 16, 64]
 
-profile_models: 
+profile_models:
   classification_malaria_v1:
     model_config_parameters:
       instance_group:
-        -
-          kind: KIND_GPU
-          count: [1,2]
+        - kind: KIND_GPU
```

Comment:
i'm not 100% sure that this is equivalent yaml

Reply:
I tried it out (in a unit test) and it worked fine

Reply:
fair enough